internal/parse/convert.go

```diff
@@ -146,6 +147,8 @@ func processTargetDetail(annotation *file.Annotation, input string) error {
 				return fmt.Errorf("%w, %v", ErrInvalidSyntax, err)
 			}
 			annotation.TargetLineTo = upperBound
+		} else {
```

Comment:
Why don't you do like this? Will there be any other conditions?

```.go
annotation.TargetLineTo = math.MaxInt32 // TODO: Consider making this Int64
if ub != "" {
// ....
}
```

Reply:
100% agreed, I was mad or something....

fa4a970